docs/UserGuide.md

```diff
@@ -55,16 +46,29 @@ and bidder exists in the property list and bidder list.
  
  All types of meeting require a valid Property Id and Bidder Id for it to be added to the schedule.
 
+💡 SELLER_ID indicates the id of the seller.
+
+💡 BIDDER_ID indicates the id of the bidder.
+
+💡 PROPERTY_ID indicates the id of the property listing.
+
+💡 VENUE indicates the venue of the meeting.
+
+💡 DATE indicates the meetingDate of the meeting.
+
+
 # Features
 
```

Comment:
We can have a pictorial representation of the UI, such as how people take screen shot and demarcate which part of the UI is for what purpose